File: TV_Transmitter.pl
#!/usr/bin/perl

use 5.022;
use warnings;

my $signal = <>;
chomp $signal;

# if ($signal ne '-1') HARDCODE
if (substr($signal, 0, 1) ne '-') {
	say $signal;
}

1;

# by Tovarisch Trunaev, 2026, Russia, Uray

File: TV_Transmitter.pas
Program TV_Transmitter;
Const 
	Stop = '-1';
Var 
    signal : string;
Begin;
    ReadLn (signal)
    IF (signal != Stop) Then
		WriteLn (signal);
end.

{ by Tovarisch Trunaev, 2026, Russia, Uray }
